modules/installation-user-infra-machines-iso.adoc

```diff
@@ -81,7 +81,7 @@ the `coreos-installer` command instead of adding kernel arguments. If you
 run the live installer without options or interruption, the installer boots up to a
 shell prompt on the live system, ready for you to install {op-system} to disk.
 
-. Review the “Advanced {op-system} bare metal installation configuration”
+. Review the “Advanced {op-system} installation reference”
```

Comment:
I noticed that we updated the title of this module but it wasn't reflected in this reference.